tests/data_loading/test_sampler.py

```diff
@@ -1,174 +1,102 @@
-import unittest
+# ToDo: Add sampler tests
 
+import unittest
 import numpy as np
-
-from delira.data_loading.sampler import LambdaSampler, \
-    PrevalenceRandomSampler, \
-    PrevalenceSequentialSampler, \
-    RandomSampler, \
-    SequentialSampler, \
-    StoppingPrevalenceRandomSampler, \
-    WeightedRandomSampler, \
-    WeightedPrevalenceRandomSampler
+from delira.data_loading.sampler import RandomSamplerWithReplacement, \
+    PrevalenceRandomSampler, SequentialSampler, \
+    RandomSamplerNoReplacement, BatchSampler
 
 from ..utils import check_for_no_backend
-
 from .utils import DummyDataset
 
 
 class SamplerTest(unittest.TestCase):
+    def setUp(self) -> None:
+        self.dset = DummyDataset(600, [0.5, 0.3, 0.2])
```

Comment:
I would like a different dataset with an internal ordering with ids, so we can check correct sampling of them.
sequential: sequential order
random: check a few samples with seed + check if all samples are sampled 
prevalence: check for average present labels for large number of samples with some boundaries

Reply:
This is the only thing I would prefer to move to a separate issue.